src/modm/communication/xpcc/dispatcher.cpp

```diff
@@ -34,6 +34,20 @@ xpcc::Dispatcher::update()
 
 	//Check if a new packet was received by the backend
 	while (this->backend->isPacketAvailable())
+	{
+		this->updateOnceRx();
+		this->backend->dropPacket();
```

Comment:
Shouldn't `backend->dropPacket();` get called inside `updateOnceRx()`?

Reply:
That's one point of doing this, to not drop the packet, to ... :

> Then it is possible to use the XPCC dispatcher and postman,
> and combine e.g. the zeromq gateway example with that.